taichi/program/context.h

```diff
@@ -88,11 +88,17 @@ struct RuntimeContext {
     set_array_device_allocation_type(arg_id, DevAllocType::kRWTexture);
   }
 
-  void set_arg_external_array(int arg_id, uintptr_t ptr, uint64 size) {
+  void set_arg_external_array(int arg_id,
+                              uintptr_t ptr,
+                              uint64 size,
```

Comment:
Was thinking this "size" a bit misleading. Looks like it means "number of elements" instead of "size in bytes"

Reply:
Good point! will likely merge this with set_arg_ndarray in a followup PR and will do the batch renames then ;)